.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💡 Feature request
+    url:     url: https://github.com/zkSNACKs/WalletWasabi/discussions/categories/ideas-contributions
```

Comment:
Why URL twice?

Reply:
url still twice maybe that is the problem

Reply:
fixed in 12a374a